test/unit/uk/gov/hmrc/tariffclassificationfrontend/forms/DecisionFormConstraintsSpec.scala

```diff
@@ -16,14 +16,15 @@
 
 package uk.gov.hmrc.tariffclassificationfrontend.forms
 
-import play.api.data.Form
 import play.api.libs.json.{JsObject, JsString, JsValue}
 import uk.gov.hmrc.play.test.UnitSpec
+import uk.gov.hmrc.tariffclassificationfrontend.service.CommodityCodeService
 
 class DecisionFormConstraintsSpec extends UnitSpec {
 
-  private val commodityCodeErrorMessage = "Format must be empty or numeric between 6 and 22 digits"
-  private val decisionForm: Form[DecisionFormData] = DecisionForm.form
+  private val decisionForm = new DecisionForm(new CommodityCodeConstraints(new CommodityCodeService))
```

Comment:
If the CommodityCodeConstraints were mocked then you could reduce the number of different failure scenarios below.  Would just need to validate that the if the (mocked) constraint says "no" then the form says "no"

Reply:
I've mocked the CommodityCodeService. But good point about the scenarios